packages/renovate-config-monorepo/package.json

```diff
@@ -1028,5 +1056,8 @@
         "https://github.com/googlechrome/workbox"
       ]
     }
+  },
+  "dependencies": {
+    "gh-got": "^8.1.0"
```

Comment:
Any idea where this came from?

Reply:
That's my bad actually given I had to install it to be able to run the command

Reply:
Will remove it